test/unit/region.spec.js

```diff
@@ -484,6 +487,38 @@ describe('region', function() {
     });
   });
 
+  describe('when showing an attached view', function() {
+    let testView;
+    let region;
+    let anotherRegion;
+    let collectionView;
+
+    beforeEach(function() {
+      this.setFixtures('<div id="reg1"></div><div id="reg2"></div><div id="cv"></div><div id="view"></div>')
+      region = new Region({ el: '#reg1' });
+      anotherRegion = new Region({ el: '#reg2' });
+      collectionView = new CollectionView({ el: '#cv' });
+      testView = new View({ el: '#view', template: _.noop });
+    });
+
+    it('should not throw an error if view is a prerendered one', function() {
```

Comment:
along with the changes above, I don't think pre-rendered will matter, so this test can be removed